openid-caep-interoperability-profile-1_0.md

```diff
@@ -175,6 +193,28 @@ Receivers MUST be prepared to accept events with any of the subject identifier f
 ## Event Signatures
 All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
 
+## OAuth Service
+
+### Authorization Server
+* SHALL distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [RFC8414]{{RFC8414}}
+* MUST support at least one of the following to obtain a short-lived access token 
```

Comment:
What is "short-lived" should we clarify / recommend some time bounds?

Reply:
FAPI mentions "minutes/hours" https://openid.bitbucket.io/fapi/fapi-2_0-security-profile.html#name-access-token-lifetimes

Reply:
@aaronpk please let me know if this is a correct way of referencing a draft spec